main/ertp/api/issuer.md

```diff
@@ -1,214 +1,275 @@
 # Issuer
 
-The issuer cannot mint new amounts, but it can create empty purses and payments. The issuer can also transform payments (splitting payments, combining
-payments, burning payments, and claiming payments exclusively). The issuer should be obtained from a trusted source and then relied upon as the
-authority as to whether an untrusted payment is valid.
+An `issuer` is the authority on what holds digital assets of its kind. 
+While it cannot create new value by creating digital assets like a `mint`, 
+it recognizes and identifies `purses` and `payments` that carry actual value.
+It can create empty `purses` and transform `payments` (by splitting, 
+combining, burning, or exclusively claiming). 
+
+An `issuer` has an unchangeable one-to-one relationship with the `mint` and
+`brand` that were created with it. For any `brand` for which you will accept
+`payments` in, you should obtain its `issuer` from a trusted source.
+You can then rely on that `issuer` as the authority to 
+determine if an untrusted `payment` of that `brand` is valid.
+
+## makeIssuerKit(allegedName, amountMathKind)
+- `allegedName` `{String}` 
+- `amountMathKind` `{MathKind}` - optional
+- Returns `{IssuerKit}`
+
+Makes a new `issuer` as well as its one-to-one associated ERTP objects; a `mint` and a `brand`. 
+All are in unchangeable one-to-one relationships with each other. 
+
+While this also returns an `amountMath`, we strongly urge that instead of using that one, which
+is asynchronous, you use `makeLocalAmountMath(issuer)` to create a local, synchronous, version
+and use that one. 
```

Comment:
The party that gets an `amountMath` by calling `makeIssuerKit()` *should* use that one, because it's synchronous for them. They just shouldn't share it outside the vat, because any code receiving it remotely will get an asynchronous presence. Outside the originating vat, you should make a local copy, for reasons explained several times.